simulator-core/src/test/java/com/evbox/everon/ocpp/it/provisioning/GetBaseReportIt.java

```diff
@@ -10,15 +10,14 @@
 
 import static com.evbox.everon.ocpp.mock.constants.StationConstants.DEFAULT_CALL_ID;
 import static com.evbox.everon.ocpp.mock.constants.StationConstants.STATION_ID;
-import static com.evbox.everon.ocpp.v20.message.station.GetBaseReportRequest.ReportBase.CONFIGURATION_INVENTORY;
-import static com.evbox.everon.ocpp.v20.message.station.GetBaseReportRequest.ReportBase.FULL_INVENTORY;
+import static com.evbox.everon.ocpp.v20.message.station.GetBaseReportRequest.ReportBase.*;
 import static com.evbox.everon.ocpp.v20.message.station.GetBaseReportResponse.Status.ACCEPTED;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 
 public class GetBaseReportIt extends StationSimulatorSetUp {
 
-        private static final int NOTIFY_REPORT_VARIABLES_LESS_ONE = 12;
+        private static final int NOTIFY_REPORT_VARIABLES_LESS_ONE = 13;
```

Comment:
can we get rid of this ugly constant somehow?

Reply:
I'm looking into it

Reply:
I didn't find any way of calculating the number of expected variables automatically

Reply:
It's a test expecting 1 variable per page in report and it's not really a good way to test pagination.
Also 1 variable per page was a temporary solution just for tests with Everon. It makes no sense to unit test such.

You can create a TD story to change pagination to some more reasonable value and close EVS-7 meanwhile.